ocaml/src/fragment_cache_test.ml

```diff
@@ -268,10 +281,55 @@ let test_long () =
     fetch 1000 >>= fun (found, missed) ->
     Lwt_io.printlf "done: found:%i missed:%i" found missed >>= fun () ->
     cache # drop ~global:true 0l >>= fun () ->
-    assert (cache # _check ());
     Lwt.return ()
   in
-  run_with_fragment_cache 20_000L _inner "test_long"
+  run_with_local_fragment_cache 20_000L _inner "test_long"
+
+let _test_lookup2 (cache : cache) =
+  let bid = 0l in
+  let key = "key" in
+  let size = 1_000_000 in
+  let value = Lwt_bytes.create size in
+  let value' = Lwt_bytes.to_string value in
```

Comment:
apparently there is, but for some reason I assumed there wasn't. adapting the code.